clients/python/moondream/cloud_vl.py

```diff
@@ -1,27 +1,31 @@
+import base64
 import json
 import urllib.request
-import base64
-from PIL import Image
 from io import BytesIO
-from typing import Union, Optional, Literal
+from typing import Literal, Optional, Union
 
+from PIL import Image
 
 from .types import (
     VLM,
     Base64EncodedImage,
     CaptionOutput,
-    EncodedImage,
-    QueryOutput,
     DetectOutput,
+    EncodedImage,
     PointOutput,
+    QueryOutput,
     SamplingSettings,
 )
 
 
 class CloudVL(VLM):
-    def __init__(self, api_key: str):
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        api_url: Optional[str] = "https://api.moondream.ai/v1",
```

Comment:
The type is no longer `Optional` right? If it's not passed in the code below won't work?